test/stringify.js

```diff
@@ -248,7 +248,7 @@ describe('Expression', function () {
                 type: doctrine.Syntax.NameExpression,
                 name: 'Number'
             }]
-        }, { topLevel: true }).should.equal('String|Number');
+        }, { topLevel: true, compact: true }).should.equal('String|Number');
```

Comment:
I have also explicitly set the `compact` flag to true on this test to assure that the compact behavior still works as expected